<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LeadDetails</class>
 <widget class="QWidget" name="LeadDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1086</width>
    <height>867</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="leadInformationGB">
     <property name="title">
      <string>Lead Information</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="modifiedDateLabel">
          <property name="text">
           <string>Date Modified:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="modifiedDate">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="modifiedByLabel">
          <property name="text">
           <string>by</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="modifiedBy">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>237</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="5">
         <widget class="QLabel" name="label_29">
          <property name="text">
           <string>Date Created:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QLabel" name="createdDate">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="7">
         <widget class="QLabel" name="createdByLabel">
          <property name="text">
           <string>by</string>
          </property>
         </widget>
        </item>
        <item row="0" column="8">
         <widget class="QLabel" name="createdBy">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <widget class="QGroupBox" name="detailsBox">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Details</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Lead Source:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="2">
            <widget class="QComboBox" name="leadSource">
             <item>
              <property name="text">
               <string/>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Cold Call</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Existing Customer</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Self Generated</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Employee</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Partner</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Public Relations</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Direct Mail</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Conference</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Trade Show</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Web Site</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Word of mouth</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Email</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Campaign</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Other</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Lead Source Description:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QTextEdit" name="leadSourceDescription"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_28">
             <property name="text">
              <string>Campaign:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="campaignName">
             <item>
              <property name="text">
               <string/>
              </property>
             </item>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_26">
             <property name="text">
              <string>Referred By:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1" colspan="2">
            <widget class="QLineEdit" name="referedBy"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>First Name:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QComboBox" name="salutation">
             <item>
              <property name="text">
               <string/>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Mr.</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Ms.</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Mrs.</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Dr.</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Prof.</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QLineEdit" name="firstName"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Last Name:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1" colspan="2">
            <widget class="QLineEdit" name="lastName"/>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Birthdate:</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1" colspan="2">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLineEdit" name="birthdate">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="calendarWidget" native="true"/>
             </item>
            </layout>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Account Name:</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1" colspan="2">
            <widget class="QLineEdit" name="accountName"/>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_25">
             <property name="text">
              <string>Title:</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1" colspan="2">
            <widget class="QLineEdit" name="title"/>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Department:</string>
             </property>
            </widget>
           </item>
           <item row="9" column="1" colspan="2">
            <widget class="QLineEdit" name="department"/>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Assigned to:</string>
             </property>
            </widget>
           </item>
           <item row="10" column="1">
            <widget class="QComboBox" name="assignedUserName">
             <item>
              <property name="text">
               <string/>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="otherDetailsBox">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Other Details</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="1">
            <widget class="QComboBox" name="status">
             <item>
              <property name="text">
               <string/>
              </property>
             </item>
             <item>
              <property name="text">
               <string>New</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Assigned</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>In Process</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Converted</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Recycled</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Dead</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Status Description:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QTextEdit" name="statusDescription"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Opportunity Amount:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="opportunityAmount"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>Office Phone:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="phoneWork"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>Mobile:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="phoneMobile"/>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_24">
             <property name="text">
              <string>Home Phone:</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLineEdit" name="phoneHome"/>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>Other Phone:</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QLineEdit" name="phoneOther"/>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>Fax:</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QLineEdit" name="phoneFax"/>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>Do Not Call:</string>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="QCheckBox" name="doNotCall">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="label_23">
             <property name="text">
              <string>Email Address:</string>
             </property>
            </widget>
           </item>
           <item row="10" column="1">
            <widget class="QLineEdit" name="email1"/>
           </item>
           <item row="11" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="11" column="1">
            <widget class="QLineEdit" name="email2"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Status:</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QGroupBox" name="addressesBox">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>Addresses</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Primary Address:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="2">
            <widget class="QLineEdit" name="primaryAddressStreet"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>City:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QLineEdit" name="primaryAddressCity"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_27">
             <property name="text">
              <string>State:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="2">
            <widget class="QLineEdit" name="primaryAddressState"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_30">
             <property name="text">
              <string>Postal Code:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1" colspan="2">
            <widget class="QLineEdit" name="primaryAddressPostalcode"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_31">
             <property name="text">
              <string>Country:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1" colspan="2">
            <widget class="QLineEdit" name="primaryAddressCountry"/>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_33">
             <property name="text">
              <string>City:</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1" colspan="2">
            <widget class="Line" name="line">
             <property name="lineWidth">
              <number>1</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="8" column="1" colspan="2">
            <widget class="QLineEdit" name="altAddressCity"/>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="label_34">
             <property name="text">
              <string>State:</string>
             </property>
            </widget>
           </item>
           <item row="9" column="1" colspan="2">
            <widget class="QLineEdit" name="altAddressState"/>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="label_35">
             <property name="text">
              <string>Postal Code:</string>
             </property>
            </widget>
           </item>
           <item row="10" column="1" colspan="2">
            <widget class="QLineEdit" name="altAddressPostalcode"/>
           </item>
           <item row="11" column="0">
            <widget class="QLabel" name="label_36">
             <property name="text">
              <string>Country:</string>
             </property>
            </widget>
           </item>
           <item row="11" column="1" colspan="2">
            <widget class="QLineEdit" name="altAddressCountry"/>
           </item>
           <item row="12" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Copy address from primary:</string>
             </property>
            </widget>
           </item>
           <item row="12" column="1">
            <widget class="QCheckBox" name="copyAddressFromPrimary">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="13" column="2">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>174</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="7" column="1" colspan="2">
            <widget class="QLineEdit" name="altAddressStreet"/>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_32">
             <property name="text">
              <string>Other Address:</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="descriptionBox">
        <property name="title">
         <string>Description:</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QTextEdit" name="description"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>504</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="saveButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>&amp;Save</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>504</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
